app/connectors/SelfAssessmentODSConnector.scala

```diff
@@ -86,25 +86,36 @@ class CachingSelfAssessmentODSConnector @Inject() (
     )
   }
 
+  private def ignoreCacheForSelfAssessment(utr: String): Boolean = {
+    val isStubbedEnvironment: Boolean = applicationConfig.environment == "local"
+    isStubbedEnvironment && utr >= "0000000010" && utr <= "0000000020"
+  }
+
```

Comment:
I don't like this. Mixing production and test code.
What about in staging binding to the connector directly without the cache?
So you have a config useCache of some sort that is overridden in staging.

Reply:
To be honest I don't think this is necessary, the cache is session based. So if you log with a new session it will get the newest data.

Reply:
I know what you mean. But I want use of the tool to be as smooth as possible. When SME changes a value he will expect the values to update immediately and not have to sign in out again. If I bind directly in staging without cache then we won't be testing the cache in that environment when doing say performance testing.

Reply:
This will not affect production or QA. Only local.

Reply:
Other solutions discussed are no better. We keep this :)